spring-cloud-netflix-sidecar/src/main/java/org/springframework/cloud/netflix/sidecar/SidecarConfiguration.java

```diff
@@ -80,13 +97,29 @@ public EurekaInstanceConfigBean eurekaInstanceConfigBean() {
 			int port = this.sidecarProperties.getPort();
 			config.setNonSecurePort(port);
 			config.setInstanceId(getDefaultInstanceId(this.env));
-			if(StringUtils.hasText(springAppName)) {
+			if (StringUtils.hasText(springAppName)) {
```

Comment:
What do you mean? Sorry I can't understand since https://github.com/spring-cloud/spring-cloud-netflix/blob/master/eclipse/eclipse-code-formatter.xml describe that `if` must be followed by space before brace